code/modules/mob/mob.dm

```diff
@@ -1006,10 +1006,18 @@
 	SHOULD_CALL_PARENT(FALSE)
 	gib()
 
-/mob/explosion_act()
+/mob/get_contained_external_atoms()
 	. = ..()
-	if(!blinded)
-		flash_eyes()
+	if(.)
+		LAZYREMOVE(., get_organs())
+
+/mob/explosion_act(var/severity)
+	. = ..()
+	if(!QDELETED(src))
```

Comment:
Is this something that can happen because an exploded object both deletes its contents, and also explosion_acts them?

Reply:
Previously things have qdeleted lower down in the chain due to calling physically_destroyed, I don't think it's likely with /mob but it's not an expensive check.

Reply:
Okay yeah went back and checked. If atom-level explosion act triggers a reagent explosion, the atom might be deleted by the time it returns control to the parent proc.